core/src/main/scala/kafka/controller/ControllerEventManager.scala

```diff
@@ -69,14 +69,16 @@ class QueuedEvent(val event: ControllerEvent,
 class ControllerEventManager(controllerId: Int,
                              processor: ControllerEventProcessor,
                              time: Time,
-                             rateAndTimeMetrics: Map[ControllerState, KafkaTimer]) extends KafkaMetricsGroup {
+                             rateAndTimeMetrics: Map[ControllerState, KafkaTimer],
+                             eventQueueTimeTimeoutMs: Long = 60000) extends KafkaMetricsGroup {
   import ControllerEventManager._
 
   @volatile private var _state: ControllerState = ControllerState.Idle
   private val putLock = new ReentrantLock()
   private val queue = new LinkedBlockingQueue[QueuedEvent]
   // Visible for test
   private[controller] val thread = new ControllerEventThread(ControllerEventThreadName)
+  val eventQueueTimeMetricTimeoutMs = eventQueueTimeTimeoutMs
```

Comment:
We dont need new `eventQueueTimeMetricTimeoutMs` variable. we can use `eventQueueTimeTimeoutMs` directly ?